### PR TITLE
Update connect chat container (part 2)

### PR DESCRIPTION
**Pull request by lisasy**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
- Rename certain class names to improve BEM syntax and naming
- Update typography of message cells
- Update header of message area
- Use new 'info' icon to expand sidebar
- Update form fields for both writing and editing a message
- Implement Crayons styling for 'delete' modal

Related next-steps & optimizations would be:
- Implement growing the height of the textfield as user enters more characters
- Improve `delete modal` experience by making overlay span entire viewport window

## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/issues/8437

#### Default
Screen Shot 2020-06-12 at 2 27 04 PM

#### Hover on message cell
Screen Shot 2020-06-12 at 2 27 11 PM

#### Edit message
Screen Shot 2020-06-12 at 2 27 16 PM

#### Delete modal
Screen Shot 2020-06-12 at 2 27 23 PM

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

Because I modified certain class names, I'll need to update tests and snapshots to reflect these changes.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by lisasy** [comment]
So, I messed up the other PR and it merged but didn't deploy due to Travis CI build errors — seems like this error was external to the PR.

I created this new PR and attempting this all over again, hoping it succeeds.

File: app/javascript/chat/chat.jsx [hunk 1]
```diff
@@ -1778,24 +1776,17 @@ export default class Chat extends Component {
           if (e.keyCode === 13) this.triggerActiveContent(e);
         }}
         tabIndex="0"
-        href={path}
+        href="#/"
```
Comment (Zhao-Andy) [inline app/javascript/chat/chat.jsx:1779]: Should this have changed?

Comment (lisasy) [reply app/javascript/chat/chat.jsx:1779]: It shouldn't have, I actually didn't edit this but when I did `git merge master`, this merge conflict appeared.

**Comment by Zhao-Andy** [comment]
> I'm confused as to why this is passing if there were changes to chat.jsx. Usually the snapshot tests should fail. Some stuff is just prettier in action, but other stuff like adding an alt to the SVG should cause the snapshot to fail.

Huh, yeah I'm actually not sure why this happened either...

**Pull request merged**